rai/node/node.cpp

```diff
@@ -3524,6 +3525,25 @@ bool rai::handle_node_options (boost::program_options::variables_map & vm)
 			result = true;
 		}
 	}
+	else if (vm.count ("wallet_import_seed"))
+	{
+		rai::raw_key key;
+		if (!key.data.decode_hex (vm["key"].as<std::string> ()))
+		{
+			inactive_node node (data_path);
+			rai::keypair gen_key;
+			auto wallet (node.node->wallets.create (gen_key.pub));
+			wallet->enter_initial_password ();
+			rai::transaction transaction (wallet->store.environment, nullptr, true);
+			wallet->change_seed (transaction, key);
+			std::cout << gen_key.pub.to_string () << std::endl;
```

Comment:
Could you use `to_account` here? Public keys are meant to be internal where possible.